src/styles.css

```diff
@@ -515,7 +537,7 @@ nav .burger-icon.burger-open {
   stroke-linejoin: round;
 }
 .temp-line-text {
-  font-size: 1rem;
+  font-size: 14pt;
```

Comment:
why not variable? why not rem units?

Reply:
Because this is for temperature SVG and it is being scaled on its own.